dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsApiExtendedServiceImpl.java

```diff
@@ -483,6 +483,27 @@ public Response setAggregatedMetrics(String id, String versionId, Partner platfo
         return Response.ok().entity(version.getMetricsByPlatform()).build();
     }
 
+    @Override
+    public Map<Partner, Metrics> getAggregatedMetrics(String id, String versionId) {
+        Entry<?, ?> entry;
+        try {
+            entry = getEntry(id, Optional.empty());
+        } catch (UnsupportedEncodingException | IllegalArgumentException e) {
+            throw new CustomWebApplicationException("Could not decode registry id", HttpStatus.SC_BAD_REQUEST);
```

Comment:
Message seems a little misleading now, could be any part of the id or the version, not just the registry (github.com, bitbucket.org) bit